### DISCUSSION TITLE
爲什麼show_notifications設定排除了設定部署消息的可能？

### DISCUSSION THREAD

**Discussion by johnmave126** [rime/home#1574] [Q&A]
根據[1055](https://github.com/rime/weasel/pull/1055)，show_notifications的設定是不對deployment生效的，而從[這段代碼](https://github.com/rime/weasel/blob/ec3294ba5f08712049f163ea13431d01f6a5034a/RimeWithWeasel/RimeWithWeasel.cpp#L766-L768)也確實可以看出對於部署的消息，設定上是永遠顯示的。
我當前做了一個計劃任務，每個小時執行一次WeaselDeploy.exe /sync，但是感覺每次都彈出部署消息有點惱人。有點好奇永遠顯示部署消息的決定是有什麼特定的原因嗎？允許show_notifications包含部署消息的設置會帶來什麼影響？

**Comment by fxliang** [comment]
因为deploy时输入法不可用，因此deploy 消息是个关键状态不应被隐藏，而且正常使用时deploy消息频度极低，强制不隐藏显示几百毫秒也不造成太大影响

就你这个特殊需求而言，fork 仓库删减这个条件让ci构建产物自用即可
